src/api/app.js

```diff
@@ -69,10 +69,7 @@ module.exports.authorise = (appToken, permissions, options) => {
 module.exports.connectAuthorised = (appToken, authUri) => {
   return getObj(appToken)
     .then((app) => app.auth.loginFromURI(authUri))
-    .then((connectedApp) => {
-      freeObj(appToken);
-      return genHandle(connectedApp);
-    });
+    .then((connectedApp) => appToken);
```

Comment:
That's right, but both instances seem to reference the same connection, we can double check with Ben, but if this is correct, I thought it's better to return just the same reference to make it simpler for the user/client app.
For example, if you authorise an app (token/instance 1), and then connect as authorised (token/instance 2), you can check using either of the two tokens 1 or 2 if it's registered (using `window.safeApp.isRegistered(token)`) and both return `true`.